community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/state/storeview/LabelScanViewIdIterator.java

```diff
@@ -34,20 +32,13 @@
  */
 class LabelScanViewIdIterator implements PrimitiveLongResourceIterator
```

Comment:
this class not can go completely since we do not need it anymore and it performs only delegation operations now, also javadoc is wrong now